Error during download phase (can't get /data/StableDiffusion/model.ckpt)

<!--  PLEASE FILL THIS OUT, IT WILL MAKE BOTH OF OUR LIVES EASIER -->

**Has this issue been opened before?**

- [x ] It is not in the [FAQ](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ), I checked.
- [ x] It is not in the [issues](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues?q=), I searched.


**Describe the bug**

`sudo docker compose --profile download up --build` fails to download StableDiffusion, investigating further up the terminal output shows the following error:

```
webui-docker-download-1  | 10/21 00:00:58 [ERROR] CUID#7 - Download aborted. URI=https://www.googleapis.com/storage/v1/b/aai-blog-files/o/sd-v1-4.ckpt?alt=media
webui-docker-download-1  | Exception: [AbstractCommand.cc:351] errorCode=24 URI=https://www.googleapis.com/storage/v1/b/aai-blog-files/o/sd-v1-4.ckpt?alt=media
webui-docker-download-1  |   -> [HttpSkipResponseCommand.cc:215] errorCode=24 Authorization failed.
```

**Which UI**

'download' profile


**Hardware / Software**
 - OS: Ubuntu
 - OS version: 20.04 LTS
 - Docker Version: <!--  get using `docker version` -->
 - Docker compose version: v2.12.0
 - Repo version: 31e4dec (latest)
 - RAM: 64 GB
 - GPU/VRAM: 24 GB

**Steps to Reproduce**
Vanilla docker and docker-compose install
Clone repo
cd to repo
`sudo docker compose --profile download up --build`

--- a/services/download/links.txt
+++ b/services/download/links.txt
@@ -1,4 +1,4 @@
-https://www.googleapis.com/storage/v1/b/aai-blog-files/o/sd-v1-4.ckpt?alt=media
+https://drive.yerf.org/wl/?id=EBfTrmcCCUAGaQBXVIj5lJmEhjoP1tgl&mode=grid&download=1
   out=StableDiffusion/model.ckpt
 https://github.com/TencentARC/GFPGAN/releases/download/v1.3.4/GFPGANv1.4.pth
   out=GFPGAN/GFPGANv1.4.pth

